src/database/models/team_item.rs

```diff
@@ -73,3 +73,31 @@ pub struct TeamMember {
     pub name: String,
     pub role: String,
 }
+
+impl TeamMember {
+    pub async fn get_from_team<'a, 'b, E>(id: TeamId,  executor: E) -> Result<Vec<TeamMember>, super::DatabaseError> where E: sqlx::Executor<'a, Database = sqlx::Postgres>,{
+        use futures::stream::TryStreamExt;
+
+        let team_members = sqlx::query!(
+            "
+            SELECT id, user_id, member_name, role
+            FROM team_members
+            WHERE team_id = $1
+            ",
+            id as TeamId,
+        ).fetch_many(executor)
+            .try_filter_map(|e| async {
```

Comment:
This should be `StreamExt::filter_map` rather than `TryStreamExt::try_filter_map` and `Ok` wrapping the result.

Reply:
The rest of the database uses the Try version, why not use it here?

Reply:
It doesn't accurately represent what the method is doing?  It's like using `Result<T, ()>` to represent an optional value rather than just using `Option<T>`.  `TryStreamExt` is an *extension* of `StreamExt`, so using the original and less complicated version is a good idea.